samples/graph-meeting-notification/nodejs/helper/certHelper.js

```diff
@@ -0,0 +1,130 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
We should rename this file as certificate-helper.

Also, update header which includes file name.

Remove extra empty line before const.